addon/components/yeti-table/component.js

```diff
@@ -207,7 +207,7 @@ class YetiTable extends DidChangeAttrsComponent {
    * the @sortable argument to all columns.
    */
   @argument('boolean')
-  sortable = true;
+  sortable = this.get('config').sortable === undefined ? true : this.get('config').sortable;
```

Comment:
I think we can just do `sortable = this.get('config').sortable;` here

Reply:
as above, wouldn't this then leave sortable defaulted to undefined if the key was not defined in thier config? This would never default to true.

Reply:
You're totally right, sorry. I must have been a bit sleepy yesterday.

Reply:
Given this one MUST be this way, the pagination one could be as you suggested, but I like them both following the same format and pagination being false instead of undefined. Let me know if you still want the pagination one changed or left as is

Reply:
Yes, I agree with you. 👍